core/src/components/button/test/form-reference/button.e2e.ts

```diff
@@ -130,6 +130,25 @@ configs({ directions: ['ltr'], modes: ['ios'] }).forEach(({ title, config }) =>
 
       expect(submitEvent).not.toHaveReceivedEvent();
     });
+
+    test('should submit the form by id when form is set async', async ({ page }) => {
+      await page.setContent(
```

Comment:
Let's add a testInfo to keep track of the issue.

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/27955/commits/c6d0651b3608086512d9302c4399e7583ac61a52